### ISSUE TITLE
Cannot get addprinter to show anything.

### ISSUE THREAD

**Issue by pnunn** [michaelrsweet/hp-printer-app#22]
**Describe the bug**
I am trying to install an HP 4050 printer on KDE Neon with absolutely no success at all.

The addprinter page never populates (just a spinner running for ever).

**To Reproduce**
1. Install the snap on kde neon as per the instructions (sudo snap install hplip-printer-app --edge)
2. navigate to localhost:8000
3. Click on Add Printer
4. Page localhost:8000/addprinter never populates (just a white page with a spinner)

**Expected behavior**
I can add a printer.

**System Information:**
 - OS: KDE Neon
 - Browser: Brave

**Additional context**
I can't add this printer through the normal add printer process either as it can never find a ppd file (either the gui or cups).

**Comment by michaelrsweet** [comment]
Can you try another browser? It *should* "Just Work™" as I'm not using any "new" HTML/HTTP stuff for the web interface, but I don't have the Brave browser to test at the moment (hoping I can just load it up on one of my existing Linux systems/VMs) and I haven't tested with it at all...

I *do* know that Safari, Firefox, Chrome, and Edge all work.

Alternately, try the following command:

```
hp-printer-app devices
```

**Comment by pnunn** [comment]
Hi michael, sorry, only just got this.

So, nothing from the command line worked either when I was trying.

What I've now done is copy the entire /etc/cups directory from a kdeneon nightly installation on a different laptop, that did find the printers OK, and they are now working on this system. But absolutely nothing I tried for a considerable time worked on here.

I just tried hplip-printer-app devices and it turned up
dnssd://EPSON%5C032WF-7620%5C032Series._pdl-datastream._tcp.local/
snmp://LJ4050

which I assume is the settings from the copied cups install.
